### ISSUE TITLE
Event scripting turned awry

### ISSUE THREAD

**Issue by CaudilloFranco** [loup99/WtWSMS#102]
When I copy and paste my stuff from Notepad++ to GitHub, it messes up the spaces and such. Look at the event troop spawning in my events as a reference. Any ideas/suggestions/help?

**Comment by rquinio** [comment]
Well you shouldn't need to copy & paste stuff between git and somewhere else.
You can directly open the files of git "working directory" in the editor, save, load the game, test, commit and push.
There is a trick to tell ck2 game to look for mod files in git, check the readme.md file in the git repo.

**Comment by CaudilloFranco** [comment]
Where is the "Working Directory" by any chance?

**Comment by loup99** [comment]
If you do the "open in explorer", you find the files in the Git repository. Then you can modify that as you wish and push it to the repository.

**Comment by CaudilloFranco** [comment]
@loup99 Thanks for the clarification. The reason why I did all those commits were because I only noticed some errors after editing the files and so that is why you see many commits for the file. As for the pull request, I didn't intend to want to merge the file myself, I want everybody to take a look at it, and when it is ready to be used, it is merged with the master file. I hope this clarifies anything that wasn't interpreted the right way.
